lib/swagger_event_schema_1.rb

```diff
@@ -39,18 +39,22 @@ def swagger_event_schema(name, skip_session_fields: false, type:, &block)
         property :client_clock_occurred_at do
           key :type, :string
           key :format, 'date-time'
-          key :description, 'The RFC 3339 section 5.6 date-time when nudge actually occurred.'
+          key :description, 'The RFC 3339 section 5.6 date-time when event actually occurred.'
         end
         property :client_clock_sent_at do
           key :type, :string
           key :format, 'date-time'
-          key :description, 'The RFC 3339 section 5.6 date-time when nudge event was sent to the server.'
+          key :description, 'The RFC 3339 section 5.6 date-time when event was sent to the server.'
         end
         property :type do
           key :type, :string
           key :description, 'The data\'s type.'
           key :enum, [type]
         end
+        property :source_uri do
+          key :type, :string
+          key :description, 'client browser location when event occurred.'
```

Comment:
may not be a browser in the future, e.g. if an app.  so maybe something a little more generic

Reply:
fixed